.github/workflows/reusable-build.yml

```diff
@@ -97,10 +97,6 @@ jobs:
             coreos_kernel () {
               coreos_version=${1}
               image_linux=$(skopeo inspect docker://quay.io/fedora/fedora-coreos:${coreos_version} | jq -r '.Labels["ostree.linux"]')
-              # Gross Pin... But workaround coreos-stable being on a broken kernel...
-              if [[ "${{ matrix.kernel_flavor }}" == "coreos-stable" ]]; then
-                image_linux="6.11.3-300.fc41.x86_64"
```

Comment:
Is this worth leaving in as a comment? Would make it easier to remember if/when we need to do this again.